client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -153,33 +150,17 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 	}
 
 	renderActionTile() {
-		const { isConnected, siteUrl, translate } = this.props;
-
-		const connectUrl = addQueryArgs(
-			{
-				url: siteUrl,
-				// TODO: add a parameter to the JPC to redirect back to this step after completion
-				// and in the redirect URL include the ?action=add_business_address parameter
-				// to actually trigger the form display action after getting back to JPO
-			},
-			'/jetpack/connect'
-		);
-		const href = ! isConnected ? connectUrl : null;
+		const { siteId, translate } = this.props;
 
 		return (
-			<Fragment>
-				{ this.renderHeader() }
-
-				<TileGrid>
-					<Tile
-						buttonLabel={ translate( 'Add a business address' ) }
-						image="/calypso/images/illustrations/business-address.svg"
-						onClick={ this.handleAddBusinessAddressClick }
-						href={ href }
-						e2eType={ 'business-address' }
-					/>
-				</TileGrid>
-			</Fragment>
+			<ConnectIntro
+				buttonLabel={ translate( 'Add a business address' ) }
+				e2eType="business-address"
+				header={ this.renderHeader() }
```

Comment:
What about hard-wiring `<FormattedHeader />` into `ConnectIntro`, and only passing a `header` and `subheader` strings (or `title` and `subtitle` -- I think I'm calling it `title` in `ConnectSuccess`)? More rigid structure, more similar to `ConnectSuccess` 😁

Reply:
Okay, it'd have to be more generic (`PropType.node`) for `subtitle` because of the damn `Fragment`s. My point still stands tho 😄

Reply:
Worthy suggestion, but I prefer having a single `PropType.node` with the header, rather than having a separate one for the title and subtitle. It also allows us to cut some code in the business address step by reusing the `renderHeader` method.

Reply:
kk